src/main/resources/jenkins/plugins/extracolumns/DisableProjectColumn/button.js

```diff
@@ -0,0 +1,12 @@
+Behaviour.specify(".ec-toggle-project", "extra-column-toggle-project", 0, function (e) {
+  e.addEventListener("click", function (ev) {
```

Comment:
Sorry about that. I think we can just skip the parameter complete, since it's unused:

```suggestion
  e.addEventListener("click", function () {
```